espaper-weatherstation.ino

```diff
@@ -292,11 +292,24 @@ void drawCurrentWeather() {
 
 }
 
+unsigned int hourAddWrap(unsigned int hour, unsigned int add) {
+  hour += add;
+  if(hour > 23) hour -= 24;
```

Comment:
Hmm, did you test with 12hour style as well?

-> https://github.com/ThingPulse/espaper-weatherstation/blob/master/settings.h#L98

Reply:
Never mind, just tested for a while and it looks good.